templates/login.html

```diff
@@ -57,6 +57,12 @@
     {% endif %}
   </div>
 
+  <div class="info text-center">
+    <br>
+    <i class="fa fa-info-circle"></i>
+      Use <a href=" {{ custom.support_service.url }}">{{ custom.support_service_name }}</a> to open tickets for support questions.
```

Comment:
Let's just make the entire HTML here customizable? So it's just we can have a *list* of announcements here. Not everyone has a suppert link...

Reply:
Maybe some `custom.help.url` and `custom.help.text`?
Even when not everyone has a "support" link, I would bet everyone has some email address people could/should ping if they have some issue/problem, right?

Reply:
I actually thought that we could default to our freshdesk support account? Hmm, but maybe my thinking is not right.
Thinking about it, I believe we concluded to have the hub representative reach out to us and not the regular user

Reply:
@GeorgianaElena yeah, i think the situation for each hub is different enough that arbitrary HTML here is nice. In the future, we could also perhaps let hub admins set announcements here via the configurator

Reply:
maybe delete this section entirely? (see my comment for the code lines bellow)

Reply:
Yep, that is the other option, just go with the ANN section, but I still think it is useful to have a "link place" just below the login button...